pandas/_libs/tslibs/conversion.pyx

```diff
@@ -52,6 +52,11 @@ DT64NS_DTYPE = np.dtype('M8[ns]')
 TD64NS_DTYPE = np.dtype('m8[ns]')
 
 
+class OutOfBoundsTimedelta(ValueError):
```

Comment:
Could we add this to the API reference in `general_utility_functions.rst`?

Reply:
also can you add a doc-string here (i guess simiilar to OOB for dates)